<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLabelStudy</class>
 <widget class="QWidget" name="QLabelStudy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="alignLeft">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>alignLeft</string>
   </property>
  </widget>
  <widget class="QLabel" name="alignRight">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>alignRight</string>
   </property>
  </widget>
  <widget class="QLabel" name="alignHCenter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>alignHCenter</string>
   </property>
  </widget>
  <widget class="QLabel" name="alignTop">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>alignTop</string>
   </property>
  </widget>
  <widget class="QLabel" name="alignButtom">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>alignButtom</string>
   </property>
  </widget>
  <widget class="QLabel" name="alignVCenter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>alignVCenter</string>
   </property>
  </widget>
  <widget class="QLabel" name="alignJustify">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>alignJustify</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
